remoteit/desktop
# [DESK-644] Single device query

### Description


Be able to query for a single device from graphql and merge with devices model

Also
- Single device refresh state button
- Launch ssh form size fix
- Connection error message layout update
- Service mini state error display
- Fixing initial setup page route
- Don't update connections on endtime changes for cli re-tries

### Test plan



1. Sign in
2. Go to a single device and select it
3. Have someone else connect to it
4. Click the refresh button
5. See them connected

### Ticket



<https://remoteit.atlassian.net/browse/DESK-644>
